packages/web-forms/src/themes/2024-light/theme.scss

```diff
@@ -14,6 +14,7 @@ $accentColor: #ffffff;
 $accentTextColor: #424242;
 $primary50: $lightBlue;
 $emphasis-high: $textColour;
+$textSecondaryColor: $hintText;
```

Comment:
This doesn't seem right?

Reply:
`--text-color-secondary` takes the value from `$textSecondaryColor`, so I have overwritten the value of `$textSecondaryColor`

See node_modules/primevue-sass-theme/themes/material/material-light/_variables.scss

Should I do something else?

Reply:
¯\_(ツ)_/¯ I was thinking not overriding it makes the most sense since the not-overridden color is the color I can actually find in Figma. But I see now that we also have a reference to a color handoff on Jun 14th at 4:17PM, which I guess is the authoritative source for this aspect of the design. Seems pretty surprising from an information design perspective to override an existing library/framework variable to get its value from a variable without any apparent reference other than that mysterious timestamp!

All glibness aside, it's fine, the color is close enough. I will just say that this sort of difficulty finding and agreeing on _actual authoritative design references_ highlights the kind of value we might get from a shared design/component library tool (whether that's [Storybook](https://storybook.js.org/) or anything else). At the very least, I hope we'll have some more coherent way to reference between design and implementation when we adopt PrimeVue design tokens.